src/header/searchBar/searchBar.ts

```diff
@@ -104,6 +104,8 @@ export class SearchBar extends DefaultControl {
   private _searchFocus (e: JQueryEventObject): void {
     if (this._map.getVenue()) {
       this._map.headerManager.showSearchResults('mainSearches', this._clickOnSearchResult.bind(this))
+    } else if ((!this._map.getVenue() && !this._map.getDirection())) {
```

Comment:
```suggestion
    } else {
```
we cannot be in direction outside venue, so i think this condition is not really needed. (take time to test with just an else)